atd-vze/src/queries/gqlAbstract.js

```diff
@@ -549,13 +572,42 @@ gqlAbstractTableAggregateName (
     });
     // Join each aggregate query into one string
     const aggregatesQueryString = aggregatesQueryArray.join(" ");
-    
+
     // Return GraphQL query
     return gql`query GetLocationStats {
       ${aggregatesQueryString}
     }`;
   }
 
+  /**
+   * Sets the options for Apollo query methods
+   * @param {string} optionType - The method in question: useQuery, useMutation, etc.
+   * @param {object} optionsObject - A key value pair with Apollo config stipulations.
+   */
+  setOption(optionType, optionsObject) {
```

Comment:
Happy to see the GraphQL abstract grow to handle more logic!

Reply:
+1